tests/cache/tests.py

```diff
@@ -1239,6 +1239,12 @@ def test_memcached_deletes_key_on_failed_set(self):
         value = cache.get('small_value')
         self.assertTrue(value is None or value == large_value)
 
+    def test_close(self):
+        # Test connections are closed when the request is complete (#5133).
+        with mock.patch.object(cache._lib.Client, 'disconnect_all', autospec=True) as mock_disconnect:
```

Comment:
If `autospec=False`, then the test can still pass even if typoed:
http://engineeringblog.yelp.com/2015/02/assert_called_once-threat-or-menace.html

eg this passes:
```python
self.assertTrue(mock_disconnect.callllllled)
```

For cases where the autospec 'gotchas' don't apply, I think it's best practice to use it, to avoid footguns. (If only since people will copy-paste elsewhere, and might not think to use `call_count` vs `called`.)